graphql/util.lua

```diff
@@ -23,6 +23,15 @@ function util.bind1(func, x)
   end
 end
 
+function util.getParentField(context, name, step_back)
+  local obj = context.objects[#context.objects - step_back]
+    if obj.__type == 'List' then
+      return obj.ofType.fields[name]
+    else
+      return obj.fields[name]
+    end
+end
```

Comment:
Maybe we could default `step_back` to 1 if it's `nil`?  Also would prefer a more succinct variable name of `count`.  There's also an indentation problem.  I can address all these post-merge if you want.